src/main/java/io/cdap/plugin/dlp/DLPTransformPluginConfig.java

```diff
@@ -186,4 +221,19 @@ public void validate(FailureCollector collector, Schema inputSchema) {
       }
     }
   }
+
+  public String getLocation() {
```

Comment:
methods that can return null (like this one) should be annotated with `@Nullable`